openapi/components/schemas/ReportJournal.yaml

```
@@ -0,0 +1,54 @@
+type: object
+properties:
+  aggregationField:
+    type: string
+    description: The report data is grouped by the aggregation field.
+    enum:
+      - product.accountingCode
+      - product.id
+      - plan.id
+  currency:
+    $ref: CurrencyCode.yaml
+  bookedFrom:
+    type: string
+    description: A year and month in `yyyy-mm` format.
```

question: should we use the same descriptions as in `openapi/paths/reports@journal.yaml` here?

I feel that these descriptions describe the expected value but not what the value is used for. I think the example and pattern kind of describes this already

From `openapi/paths/reports@journal.yaml`:
```suggestion
    description: Month from which revenue is booked. Omit this value to retrieve booked revenue starting from the first booked amount.
```

---

I don't think that the descriptions need to match exactly. One is a control field, the other is informational.